aip/general/0193.md

```diff
@@ -75,132 +86,191 @@ following:
+error, the service **must** populate the `message` string in
+[`google.rpc.Status`][Status] as follows:
 
-#### ErrorInfo
+- present a developer-facing, human-readable "debug message" present
+- the message in the service's native language explain both the error
+- and offer an actionable resolution to it ([citation][Error model])
 
-The [`ErrorInfo`][ErrorInfo] message is the required way to send a
-machine-readable identifier. All error responses **must** include an
-`ErrorInfo` payload in the `details` field. Variable information
-**should** be included in the `metadata` field on `ErrorInfo` and
-**must** be included if it appears within an error message.
+**Note:** Sometimes a service will elect to always present
+`Status.message` in English rather than the application's native
+language so that messages are easily searchable in common knowledge
+bases, such as StackOverflow™.
 
-#### Uniqueness
+Important points:
 
-Each type of detail payload **must** be included at most once. For
-example, there **must not** be more than one [`BadRequest`][BadRequest]
-message in the `details` field, but there **may** be a `BadRequest` and
-a [`PreconditionFailure`][PreconditionFailure].
+- This `message` string is considered a problem description. It is
+  intended for human developers to understand the problem and is more
+  detailed than [`ErrorInfo.reason`][ErrorInfo-reason], discussed in the
+  next section.
+- Use simple descriptive language that is easy to understand (without
+  technical jargon) to clearly state the problem that results in an
+  error.
+- For pre-existing (brownfield) APIs, the information in
+ `google.rpc.Status` already exists and can’t be changed. In this
+ case, do not edit anything in this `google.rpc.Status`, because some
+ users might have automation running against the pre-existing
+ content.  For example, any changes in text for the `message` string
+ could cause breakages. For more information, see [Changing Error
+ Messages](#changing-error-messages).
 
-**Note:** `ErrorInfo` represents a special case. There **must** be exactly one
-`ErrorInfo`. It is required.
 
-### Error messages
+##### [google.rpc.Status payload][Status]
 
-For each error, the service **must** populate the `message` field on
-[`google.rpc.Status`][Status]. This error message,
+Details of this object are summarized in the following fields, field
+descriptions, and examples:
 
-- is a developer-facing, human-readable "debug message"
-- is presented in the service's native language
-- both explains the error and offers an actionable resolution to it
-  ([citation](https://cloud.google.com/apis/design/errors#error_model))
 
-**Note:** Sometimes a service will elect to always present
-`Status.message` in English rather than the application's native
-language so that messages are easily searchable in common knowledge
-bases, such as StackOverflow&trade;.
+- `code int32`
+  - The status code, which **should** be an enum value of
```

Comment:
Why is this a **should** and not a **must**?

Reply:
The language needs to be reviewed by Stanis.

Reply:
Agree with @shwoodard that this should be **must**.

Reply:
Updated. I left the Localization text as **should** be in English. Please advise.

Reply:
SGTM